### ISSUE TITLE
didn't understand the method's goal

### ISSUE THREAD

**Issue by graddwana** [SwinTransformer/Feature-Distillation#24]
Hello everyone, I need to know more about the circumstances in which this approach has to be used.

Is it correct to say that this approach works best when our instructor model is significantly more intricate than the student model? In this instance, the student model's accuracy could be equivalent with lesser parameters.

Or do we have to stick with the same architecture for the instructor and student models? However, in this instance, I fail to see why fine-tuning should not be done directly utilizing the pre-trained teacher network.
